lang/llvm-9.0/Portfile

```diff
@@ -328,13 +328,7 @@ compiler.blacklist-append macports-clang-3.3 macports-clang-3.4
 # Override the normal compiler fallback list entirely since we have
 # such specific requirements.
 compiler.fallback   clang
-# Use each macports-clang version only if already installed, to avoid
-# dependency cycles or chaining (8.0 depends on 7.0 depends on 6.0...)
-foreach ver {8.0 7.0 6.0 5.0} {
-    if {[file exists ${prefix}/bin/clang-mp-${ver}]} {
-        compiler.fallback-append macports-clang-${ver}
-    }
-}
+
```

Comment:
If clang is the only compiler in the fallback list, and this is blacklisted on some systems (<602 and <700 are used in some of the subports) then what now happens on these systems ?

Reply:
Ok, i see macports-clang-3.7 is subsequently added to the fallback list. If this is really usable in all places clang is blacklisted, then fine.

Reply:
This was already the behaviour when no macports-clang was installed, so if there were problems they should have already been apparent.